test/opt/pass_merge_return_test.cpp

```diff
@@ -534,17 +534,17 @@ TEST_F(MergeReturnPassTest, StructuredControlFlowAddPhi) {
 ; CHECK: [[true:%\w+]] = OpConstantTrue
 ; CHECK: OpFunction
 ; CHECK: [[var:%\w+]] = OpVariable [[:%\w+]] Function [[false]]
-; CHECK: OpSelectionMerge [[dummy_loop_merge:%\w+]]
```

Comment:
As in all of the merge return stuff, "single_case_switch_merge" would be better.

Reply:
done